app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -148,7 +149,7 @@ public void onClick(View v) {
                     }
                 } else {
                     if (scienceLab.isConnected()) {
-                        quantity.setText(String.valueOf(scienceLab.getVoltage(knobMarker[knobState], 1)));
+                        quantity.setText(String.format(Locale.ENGLISH,"%.2f",scienceLab.getVoltage(knobMarker[knobState], 1)));
```

Comment:
@abhinavraj23 if you'd have reformatted the code as you have ticked the point in check list, there would be spaces between commas here.